profiles/schema.py

```diff
@@ -120,7 +121,8 @@ class Meta:
         fields = ("address_type", "primary", "email")
 
 
-class ProfileType(DjangoObjectType):
+@key(fields="id")
+class ProfileNode(DjangoObjectType):
```

Comment:
This naming is more like a convention?

Reply:
yeah, Tuomas and me been bumping into this when googling around. Usually the class or entity gets named after its interface, which are usually nodes. only if it does not `implement` anything, then it is a `Type`. so, Tuomas started followng this in kukkuu and I thin k we should try to follow it everywhere, and I will try to update the GraphQL guidelines.